cipher/transposition/transposition.go

```diff
@@ -8,19 +8,15 @@
 package transposition
 
 import (
+	"fmt"
 	"sort"
 	"strings"
 )
 
-type NoTextToEncryptError struct{}
-type KeyMissingError struct{}
+var ErrNoTextToEncrypt = fmt.Errorf("no text to encrypt")
+var ErrKeyMissing = fmt.Errorf("missing Key")
```

Comment:
```suggestion
var ErrNoTextToEncrypt = errors.New("no text to encrypt")
var ErrKeyMissing = errors.New("missing Key")
```

There is a technical overhead in using `fmt` package for errors which can be avoided if there is nothing contextual in the error by creating it with the `errors` package.